src/server/pfs/cmds/mount_unix.go

```diff
@@ -129,10 +130,24 @@ func mountCmds() []*cobra.Command {
 			if err != nil {
 				return err
 			}
+			defer c.Close()
+
+			user, err := c.WhoAmI(c.Ctx(), &auth.WhoAmIRequest{})
+			if err != nil {
+				if strings.Contains(fmt.Sprintf("%s", err), "the auth service is not activated") {
```

Comment:
Maybe try using `errors.Is`
```suggestion
				if errors.Is(err, auth.ErrNotActivated) {
```